source/patterns/@aws-solutions-constructs/aws-sns-sqs/lib/index.ts

```diff
@@ -224,6 +225,7 @@ export class SnsToSqs extends Construct {
       maxReceiveCount: props.maxReceiveCount,
       enableEncryptionWithCustomerManagedKey: activeKeys.encryptQueueWithCmk,
       encryptionKey: activeKeys.queueKey,
+      constructQueueProps: activeKeys.constructProps
```

Comment:
nit: Is the `ConstructKeys` interface the best place to put `constructProps`? It seems that `constructProps` of type `sqs.QueueProps`, and written as `activeKeys.constructProps` stands out a little in an interface that mostly consists of encryption/key-related props. Not a blocker but stuck out to me a little.